awsapilib/console/console.py

```diff
@@ -489,6 +489,13 @@ def get_mfa_type(self, email):
         mfa_type = response.json().get('mfaType')
         return None if mfa_type == 'NONE' else mfa_type
 
+    def _process_after_captcha(self, parameters, response, session=None):
```

Comment:
Maybe we can rename it to something a bit more specific? Maybe `_process_after_login_captcha`?